src/providers/indexer.ts

```
@@ -0,0 +1,784 @@
+                                newVtxos: (data.result.newVtxos || []).map(
+                                    convertVtxo
+                                ),
+                                spentVtxos: (data.result.spentVtxos || []).map(
+                                    convertVtxo
+                                ),
+                                tx: data.result.tx,
+                                checkpointTxs: data.result.checkpointTxs,
+                            };
+                        }
+                    } catch (err) {
+                        throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)